doc/tier2.md

```diff
@@ -144,6 +144,49 @@ spec:
       credentials: ecs-credentials
 ```
 
+#### Update ECS Credentials
+
+There might be an operational need to update ECS credentials on a regular basis, where the following steps could be taken:
+
+1. In case Pravega Cluster has not yet been deployed, replace "secretKey" and/or "identity" with new values inside the Pravega Yaml (or its equivalent)
```

Comment:
Do we really need this section taking into account that Update ECS Credentials makes sense only for already deployed systems?

Reply:
Removed.